server/test/handlers/inputSchedule.test.js

```diff
@@ -0,0 +1,100 @@
+        const testWorkerId = 8000000;
+        const testSchoolId = 1;
+        const testSlotId = 1;
+
+        // Act
+        const isSuccessfullyInserted = await timeslotHandler.addWorkerTimeslot(testWorkerId, testSchoolId, testSlotId);
+
+        // Assert
+        expect(isSuccessfullyInserted).toBe(true);
+    });
+});
+
+
+// describe('testing invalid user schedule input', () => {
+//     beforeEach(() => {
+//         jest.resetModules(); // Clears any cache between tests.
+//     });
+
+//     test('rejection of null values for worker, school, slot ID as well as status', async () => {
+//         // Arrange
+//         const testWorkerId = null;
+//         const testSchoolId = null;
+//         const testSlotId= null; 
+//         const testStatus=null;
+
+//         // Act
+//         const isSuccessfullyInserted = await timeslotHandler.addWorkerTimeslot(testWorkerId, testschoolId, testSlotId, testStatus);
+
+//         // Assert
+//         expect(isSuccessfullyInserted).toBe(false);
+//     });
+
+//     test('rejection of empty values for worker and school ID', async () => {
+//         // Arrange
+//         const testWorkerId = '';
+//         const testSchoolId = '';
+//         const testSlotId= ''; 
+//         const testStatus='';
+
+//         // Act
+//         const user = await timeslotHandler.addWorkerTimeslot(testWorkerId, testSchoolId,testSlotId, testStatus);
+
+//         // Assert
+//         expect(user).toBe(null);
+//     });
+
+//     test('rejection of invalid worker and school ID', async () => {
+//         // Arrange
+//         const testWorkerId = 'Invalid Worker ID';
+//         const testSchoolId = 'Invalid School ID';
+//         const testSlotId= 'Invalid Slot ID'; 
+//         const testStatus='Invalid Status';
+//         // Act
+//         const user = await workerTimeslotHandler.addWorkerTimeslot(testWorkerId, testSchoolId,testSlotId,testStatus);
+
+//         // Assert
+//         expect(user).toBe(null);
+//     });
+
+//     test('rejection of worker ID', async () => {
+//         // Arrange
+//         const testWorkerId = 'Invalid Worker ID';
+//         const testSchoolId = '001';
+//         const testSlotId= 'Invalid Slot ID'; 
+//         const testStatus='Invalid Status';
+
+//         // Act
+//         const user = await timeslotHandler.addWorkerTimeslot(testWorkerId, testSchoolId);
+
+//         // Assert
+//         expect(user).toBe(null);
+//     });
+
+//     test('rejection of school ID', async () => {
+//         // Arrange
+//         const testWorkerId = '100789';
+//         const testSchoolId = 'Invalid School ID';
+
+//         // Act
+//         const user = await timeslotHandler.addWorkerTimeslot(testWorkerId, testSchoolId);
+//         // Assert
+//         expect(user).toBe(null);
+//     });
+// });
+
+// describe('testing valid user validation', () => {
+//     beforeEach(() => {
+//         jest.resetModules(); // Clears any cache between tests.
+//     });
+// });
+
```

Comment:
was this test code, if so, comment at top that this is test code

Reply:
@Kyle199910 this test could have been used for Peter's test

Reply:
Right, excellent point, I also agree with Kyle's concern. I would recommend adding a brief comment at the top signifying that this is test code that can be leveraged once the logic is fully implemented for this functionality.

Reply:
@Kyle199910 @catai9 thank you for your response! A comment has been made!